src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/multi-cluster/multi-cluster-form/multi-cluster-form.component.html

```diff
@@ -0,0 +1,150 @@
+<cd-modal [modalRef]="activeModal">
+  <ng-container i18n="form title"
+                class="modal-title">Connect Cluster
+  </ng-container>
+  <ng-container class="modal-content">
+    <form name="remoteClusterForm"
+          #frm="ngForm"
+          [formGroup]="remoteClusterForm">
+      <div class="modal-body">
+        <cd-alert-panel *ngIf="connectionMessage !== 'Connection Verified Successfully' && connectionMessage !== ''"
+                        type="error"
+                        spacingClass="mb-3"
+                        i18n>{{ connectionMessage }}
+        </cd-alert-panel>
+        <cd-alert-panel *ngIf="connectionMessage === 'Connection Verified Successfully'"
+                        type="success"
+                        spacingClass="mb-3"
+                        i18n>{{ connectionMessage }}
+        </cd-alert-panel>
```

Comment:
I think can be have this message inline to the verify connection button? 

[Verify Connection]  Connecting [ spinner]

[Verify Connection(]button hidden when verified)   Connection verified [Tick green icon]

[Verify Connection] Connection not verified [Cancel red icon]